projects/1.21/assets/data-essence/datanessence/lang/zh_cn.json

```
@@ -0,0 +1,686 @@
+  "data_tablet.pages.buffers.page1.text": "我们的机器的设计方式不接受漏斗、管道等平凡的输入手段，而是必须将输入送至缓存器——专门将其中物品、流体、源质送至其上机器的特制容器。它们可从任意一面接受输入。机器下方最多可放置5个缓存器，再往下放置则不会起效。\n\n为方便理解，后页有一则使用示例。",
+  "data_tablet.pages.drying_table.flavor": " ◇ 不是筛子",
+  "data_tablet.pages.drying_table.name": "干燥台",
+  "data_tablet.pages.drying_table.page1.text": "某些流体在经干燥后会蒸发，从而留下其他材料；也有些会浸入物品，从而形成新物品。这些材料通常很有用，所以我们设计了能加快这一干燥过程的方法。送入流体和可选额外物品，干燥台即会用源质照射原材料，大大加快了干燥的过程。",
+  "data_tablet.pages.enticing_lure.name": "引诱饵架",
+  "data_tablet.pages.enticing_lure.page1.text": "放牧动物可能并不会让人心旷神怡，但却不可或缺。引诱饵架利用了绿宝石的迷人本性大幅降低了放牧的麻烦，当然，需要在饵架上放置一个合适的物品。需引诱的生物喜欢的食物有可能是对它们的有效饵剂。此设备不需要源质；构造体自身的性质就已足够。",
+  "data_tablet.pages.entropic_processor.flavor": " ◇ 24轮粉碎冠军",
+  "data_tablet.pages.entropic_processor.name": "熵增加工器",
+  "data_tablet.pages.entropic_processor.page1.text": "钻石是世界上数一数二的坚硬材料。利用这种坚硬性质最理智的方法，是用来切割，熵增加工器则是成果。这台机器有三套八个一组的钻石切割轮，能把掉进去的东西破碎、磨碎、粉碎，乃至打成齑粉。",
+  "data_tablet.pages.entropic_processor.page2.text": " ◇ 不建议在切割轮转动的过程接触它们。",
+  "data_tablet.pages.entropic_processor.page4.text": "许多花卉和部分其他物品可在其中制成染料，产量高于手动磨粉。",
+  "data_tablet.pages.essence_breaker.flavor": " ◇ 驱动全世界的圆石生成器",
+  "data_tablet.pages.essence_breaker.name": "方块的破坏",
+  "data_tablet.pages.essence_breaker.page1.text": "源质可以用来驱动钻头，以此挖掘其前方的任意方块。源质破坏器的设计没有特殊之处，但正因其简单，它无法仅凭自己处理掉落的物品。",
+  "data_tablet.pages.essence_breaker.page2.text": " ◇ 可通过红石禁用。",
+  "data_tablet.pages.essence_furnace.flavor": " ◇ 铜质对流烧炼器",
+  "data_tablet.pages.essence_furnace.name": "源质熔炉",
+  "data_tablet.pages.essence_furnace.page1.text": "烧炼是一项常见加工手段，但普通熔炉太过低效和缓慢，不适合我们的大多数加工流程。我们很快发现让源质为加热元件注能的方法，这一方案也能烧炼物品，从而弥补了前文的两项缺点。",
+  "data_tablet.pages.essence_furnace.page2.text": " ◇ 工作速度比普通熔炉快25%。",
+  "data_tablet.pages.fluid_acquisition.flavor": " ◇ 100%吸收力",
+  "data_tablet.pages.fluid_acquisition.name": "流体的收集",
+  "data_tablet.pages.fluid_acquisition.page1.text": "为实际将流体收集并传输到机器里，可使用源质驱动泵，让泵收集其面朝方向的一桶流体。自然，可简单将此过程反转，从而制成倒出流体的同类机器。",
+  "data_tablet.pages.fluid_acquisition.page2.text": " ◇ 必须使用导向器选取“泵”面。",
+  "data_tablet.pages.fluid_bottler.name": "流体装瓶器",
+  "data_tablet.pages.fluid_bottler.page1.text": "某些时候需要将流体自动装入铁桶等容器。源质可以简化此过程。",
+  "data_tablet.pages.fluid_mixer.name": "流体混合器",
+  "data_tablet.pages.fluid_mixer.page1.text": "流体混合器是一台简单的机器：源质驱动发动机，达到一定转速后将物品和流体混合为均一稳定的新流体。我们已经发现了它在量产熔岩方面的用途。",
+  "data_tablet.pages.fluid_mixer.page2.text": " ◇ 必须插入装载数据的数据驱动器以指定合成配方。",
+  "data_tablet.pages.fluid_tank.flavor": " ◇ 未雨绸缪",
+  "data_tablet.pages.fluid_tank.name": "流体的存储",
+  "data_tablet.pages.fluid_tank.page1.text": "这件特制的储罐是在合理大小的空间内存储流体的简单方式，容量可观。可通过铁桶手动倒入。",
+  "data_tablet.pages.fluidworks.name": "流体的传输",
+  "data_tablet.pages.fluidworks.page1.text": "流体的操纵，尤其是在机器间输送流体，是我们工作中的重要流程。从对源质表现极好的已有设计出发，我们设计了流体节点和相应导线。它们的工作方式正如预期：能点对点传输任意流体。",
+  "data_tablet.pages.fluidworks.page2.text": " ◇ 节点的连接顺序决定了传输方向：先选中出发点，后选中到达点。\n ◇ 中转节点不会发出或接收。",
+  "data_tablet.pages.industrial_infuser_processing.flavor": " ◇ 为了好玩往物品里灌能量，还整出了些有用的东西",

```

https://english.stackexchange.com/question/323606/whats-the-meaning-of-phrase-for-fun-and-profit
> the phrase "for fun and profit" generally comes up in the context of article of books devoted to transforming an interest or hobby into a source of revenue.

比较恼火的是我没搜到现成的翻译案例，除了几个十分暴力的译法（

大概可以意会一下弄成这样？
```suggestion
  "data_tablet.pages.industrial_infuser_processing.flavor": " ◇ 能量灌注：从爱好到事业",
```